pkg/cmd/root/root.go

```diff
@@ -41,6 +42,7 @@ func NewRootCommand(f *factory.Factory, version string) *cobra.Command {
 	cmd.Version = version
 
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	_ = fs.Parse(os.Args[1:])
```

Comment:
What this does?

Reply:
Parses a command line into the defined flags.